docs/_attic/glossary_entries.py

```diff
@@ -475,6 +481,20 @@
 	pronunciation='', 
 	seealso="[spot instance]")
 
+PrimaryDescriptorFile = GlossEntry("primary descriptor file",
+	acronym_full="",
+	definition="The main [descriptor file] that describes a workflow or tool.",
```

Comment:
main and primary seem like synonyms that aren't really shedding much light here

Reply:
maybe something like "the initial descriptor file that Dockstore starts from to find other descriptors"